docs/semgrep-secrets/view-triage.md

```diff
@@ -59,27 +59,23 @@ You can triage findings in bulk by performing the following steps:
 
 1. Begin by ensuring that you display all **Open** findings.
 2. Apply filters with as much specificity as possible. You may have to perform bulk triage several times. By starting with the most specific cases, and closing the findings from those specific cases, you are able to narrow down findings as you work from specific to broad filter criteria.
-3. Click the bulk select check ox.
```

Comment:
:ox: 🤣

Reply:
🐂 This must pre-date Vale. Vale is pretty good at catching these things.